python/nav/smidumps/alcatel_ind1_port_mib.py

```diff
@@ -0,0 +1,8458 @@
+# python version 1.0						DO NOT EDIT
+#
+# Generated by smidump version 0.4.8:
+#
+#   smidump -f python ALCATEL-IND1-PORT-MIB
+
+FILENAME = "SNMP/ALCATEL-IND1-PORT-MIB.mib"
+
+MIB = {
+    "moduleName" : "ALCATEL-IND1-PORT-MIB",
```

Comment:
Again, due to the changes in #1924, you should rename this entire file to `ALCATEL-IND1-PORT-MIB.py`.